app/controllers/tag_controller.rb

```diff
@@ -525,6 +525,17 @@ def comments
     render 'show'
   end
 
+  def subtopics
+    # here, we look for this special tag marking a node as having a parent
+    # collect the "slugs" i.e. the unique part of the URL, which should correspond to a tagname
+    subtopic_names = Tag.find_by(name: "parent:#{params[:topic]}")
+      &.nodes
+      &.collect(&:slug)
+    @subtopics = Tag.find_by(name: subtopic_names)
```

Comment:
I was trying this locally and this line returns nil,,, the `subtopic_names` variable above returns  slugs of the nodes' title downcase & hyphenated -- from the comment seems we are expecting this to be tagname slugs that maybe the issue